Doc/library/sqlite3.rst

```diff
@@ -518,22 +518,19 @@ Connection Objects
 
    .. method:: create_collation(name, callable)
 
-      Creates a collation with the specified *name* and *callable*. The callable will
-      be passed two string arguments. It should return -1 if the first is ordered
-      lower than the second, 0 if they are ordered equal and 1 if the first is ordered
-      higher than the second.  Note that this controls sorting (ORDER BY in SQL) so
-      your comparisons don't affect other SQL operations.
+      Create a collation named *name* using the collating function *callable*.
+      *callable* is passed two :class:`string <str>` arguments,
+      and it should return an :class:`int <integer>`:
```

Comment:
I think the angle-brackets syntax works the other way around!

```suggestion
      and it should return an :class:`integer <int>`:
```

Reply:
Darn 😆 Thanks!